src/resources/primitive_kv_flash_posix.cc

```diff
@@ -0,0 +1,125 @@
+// Copyright (C) 2021 Toitware ApS.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; version
+// 2.1 only.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// The license can be found in the file `LICENSE` in the top level
+// directory of this repository.
+
+#include "../top.h"
+
+#if defined(TOIT_LINUX) || defined(TOIT_BSD)
```

Comment:
How about Windows?

Reply:
Good point, maybe I should rename the file. `_desktop.toit`?